tests/test_0x.py

```diff
@@ -0,0 +1,129 @@
+from mev_inspect.schemas.swaps import Swap
+from mev_inspect.swaps import get_swaps
+from mev_inspect.schemas.traces import Protocol
+from mev_inspect.classifiers.trace import TraceClassifier
+from tests.utils import load_test_block
+
+
+def test_fillLimitOrder_swap():
+
+    transaction_hash = (
+        "0xa043976d736ec8dc930c0556dffd0a86a4bfc80bf98fb7995c791fb4dc488b5d"
+    )
+    block_number = 13666312
+
+    swap = Swap(
+        abi_name="INativeOrdersFeature",
+        transaction_hash=transaction_hash,
+        block_number=block_number,
+        trace_address=[0, 2, 0, 1],
+        contract_address="0xdef1c0ded9bec7f1a1670819833240f027b25eff",
+        from_address="0x00000000000e1d0dabf7b7c7b68866fc940d0db8",
+        to_address="0xdef1c0ded9bec7f1a1670819833240f027b25eff",
+        token_in_address="0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
+        token_in_amount=35000000000000000000,
+        token_out_address="0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+        token_out_amount=143949683150,
+        protocol=Protocol.zero_ex,
+        error=None,
+    )
+
+    block = load_test_block(block_number)
+    trace_classifier = TraceClassifier()
+    classified_traces = trace_classifier.classify(block.traces)
+    result = get_swaps(classified_traces)
+
+    assert swap in result
```

Comment:
how many times is swap in result? should assert that it shows up once

Reply:
also maybe assert that it's the only zero x protocol response (aren't multiple for different signatures)